### DISCUSSION TITLE
Default state of external in Tiptap LinkControl

### DISCUSSION THREAD

**Discussion by skyt-a** [mantinedev/mantine#4370] [Feature requests]
I am using a Tiptap-based rich text editor provided by Mantine and I am using [Link extension of Mantine](https://mantine.dev/others/tiptap/#link-extension).
When creating a link in LinkControl, I would like it to default to an external link, is it possible to support this feature?
I have read [the source code](https://github.com/mantinedev/mantine/blob/8c12a76c56da51af34213f18dd67c8b72a0ddb44/src/mantine-tiptap/src/controls/LinkControl/LinkControl.tsx#L53) and it would be great if we could pass the initial value of this useState via props!

**Comment by rtivital** [comment]
You are welcome to submit a PR with this feature

**Reply by skyt-a** [reply]
I created PR #4373
